util/config.cc

```diff
@@ -37,6 +38,7 @@ Config::Config(const std::string& disabled_file,
       dump_metrics_(dump_metrics),
       dump_subscriptions_(dump_subscriptions),
       log_verbosity_(log_verbosity),
+      send_in_parallel_(send_in_parallel),
```

Comment:
I don't have a better suggestion right now, but just an observation: it seems like a lot of mostly boiler plate changes to add a config setting.